core/benches/validation.rs

```diff
@@ -181,46 +194,11 @@ fn sign_blocks(criterion: &mut Criterion) {
     println!("Success count: {success_count}, Failures count: {failures_count}");
 }
 
-fn validate_blocks(criterion: &mut Criterion) {
```

Comment:
I'd create a ticket for a block validation benchmark. We need this one badly. Removing it should not be done without promising to replace it with something.

Reply:
this is now part of `chain_blocks` benchmark. Block is validated when it is being chained. This step cannot be tested separately

Reply:
This could be a problem.